File: src/data/courses/dsa/two-pointers-sliding-window/sliding-window-fixed-size/first-negative-window-k/code/cpp/first_negative_window_k_sol1.cpp
vector<int> firstNegative(vector<int>& nums, int k) {
    vector<int> result;
    for (int i = 0; i <= (int)nums.size()-k; i++) {
        int found = 0;
        for (int j = i; j < i+k; j++)
            if (nums[j] < 0) { found = nums[j]; break; }
        result.push_back(found);
    }
    return result;
}

File: src/data/courses/dsa/two-pointers-sliding-window/sliding-window-fixed-size/first-negative-window-k/code/java/FirstNegativeWindowKSol1.java
class Solution {
    public int[] firstNegative(int[] nums, int k) {
        int[] res = new int[nums.length-k+1];
        for (int i=0; i<=nums.length-k; i++) {
            for (int j=i; j<i+k; j++) if (nums[j]<0) { res[i]=nums[j]; break; }
        } return res;
    }
}